api/src/pcapi/core/offers/api.py

```diff
@@ -760,9 +766,7 @@ def edit_stock(
     """
     validation.check_stock_is_updatable(stock, editing_provider)
 
-    old_price = stock.price
-    old_quantity = stock.quantity
-    modifications: dict[str, typing.Any] = {}
+    modifications = {}
```

Comment:
Il faut garder le typage pour éviter les `# type: ignore` ensuite
```suggestion
    modifications: dict[str, typing.Any] = {}
```